cloudify_starlingx/workflows/discover.py

```diff
@@ -133,13 +136,22 @@ def generate_deployment_id(subclouds_name):
     controller_node_instance = get_controller_node_instance(
         node_instance_id, node_id, ctx=ctx)
 
+    parent_deployment_capabilities = get_parent_deployment_capabilities(
+        deployment=get_deployment(ctx.deployment.id))
+    auth_url = parent_deployment_capabilities['wrcp-ip']
```

Comment:
those fields are always in the capabilities?

Reply:
Yes